feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in feature branch. 
+
+One more change in feature 1 branch.
```

Comment:
It's really good that you have decided tot add this line of code.

Reply:
Thanks sandy.